grid/grid.css

```diff
@@ -0,0 +1,49 @@
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
```

Comment:
changes: remove CSS that are not related to grid component, like here container, component-container, component-details used commonly for all the components, so these classes can be in utility.css and import that CSS file to every component CSS files.